taui/.nvmrc

```diff
@@ -0,0 +1 @@
+v12.22.5
```

Comment:
It doesn't look to me like these match the versions inside the Docker container; here's what I'm seeing in the container:
```
ddohler@lima:~/Development/projects/echo-locator$ docker-compose run --rm --entrypoint bash taui
root@65f554bdb4f6:/usr/local/src# node --version
v12.22.7
root@65f554bdb4f6:/usr/local/src# yarn --version
1.22.15
```

Does that match what you see if you follow those steps? If so, I'd update this and `.yvmrc` so that the Docker environment and on-host environment match as closely as possible. If this already matches the versions that you have inside of Docker locally then it's more likely a problem on my end because I may have older images lying around, so this is fine as-is in that case.

Reply:
Yes, the values I added to the README match the ones used in my Docker container.